### DISCUSSION TITLE
docker部署的doris备份至obs远程仓库失败，备份状态在upload时候报错。

### DISCUSSION THREAD

**Discussion by IT-Forerunner** [apache/doris#15435] [A - General / Q&A]
场景：同一服务器内，部署了两套doris，一套为docker部署（版本1.0.3），一套为服务器本地部署（版本1.2.0）。使用备份恢复的方式进行数据迁移。
问题：docker部署的doris备份无法上传至远端仓库（s3:obs对象存储）,备份进度会卡在UPLOADING状态（State），且be1会报错，报错内容如下：
Resolved remote host IP address: xxx.xx.xxx.xx
Request ID: 
Exception name: 
Error message: curlCode: 77, Problem with the SSL CA cert (path? access rights?)
0 response headers:
W1223 11:47:25.567023   748 logging.h:72] [AWSClient] If the signature check failed. This could be because of a time skew. Attempting to adjust the signer.
W1223 11:47:25.567061   748 logging.h:72] [AWSClient] Request failed, now waiting 3200 ms before attempting again.
E1223 11:47:28.802258   748 logging.h:69] [CurlHttpClient] Curl returned error code 77 - Problem with the SSL CA cert (path? access rights?)
E1223 11:47:28.802520   748 logging.h:69] [AWSClient] HTTP response code: -1

但是同服务器本地部署的doris1.2.0备份，上传，恢复无问题（使用同一远端仓库obs）。
请教一下各位技术大牛，这个报错是哪一个环境处理问题呢，是少什么环境吗。

**Comment by zy-kkk** [comment]
endpoint开头加上http://试一下

**Reply by IT-Forerunner** [reply]
> endpoint开头加上[http://试一下](http://%E8%AF%95%E4%B8%80%E4%B8%8B)

就是已http开头的，两个doris在同一台服务器，使用同一个远程仓库（obs）,建仓库的语句也一样。docker部署的就无法上传。本地包部署的就可以。obs仓库的通的。两个doris的差异就在版本号上，和环境不同，一个在容器里（不成功的），一个在本地服务器是成功的。现在doris不成功报错，看不出是哪里有问题。

**Reply by zy-kkk** [reply]
那可能是docker部署的Doris环境里没有CA证书，尝试安装一下
`sudo yum install ca-certificates`
`service php-fpm restart`
